cmd/podman/images/list.go

```diff
@@ -184,14 +184,16 @@ func writeTemplate(imageS []*entities.ImageSummary) error {
 }
 
 func tokenRepoTag(tag string) (string, string) {
-	tokens := strings.SplitN(tag, ":", 2)
+	tokens := strings.Split(tag, ":")
 	switch len(tokens) {
 	case 0:
 		return tag, ""
 	case 1:
 		return tokens[0], ""
 	case 2:
 		return tokens[0], tokens[1]
+	case 3:
+		return tokens[0] + ":" + tokens[1], tokens[2]
```

Comment:
Is this correct?  To be safe this should be 

return tokens[0] + ":" + tokens[1], strings.Join(tokens[2:],":")

Reply:
I would figure len == 0 is an error. 1, and 2 are fine and then > 2 would be the default.

Reply:
> Is this correct? To be safe this should be
> 
> return tokens[0] + ":" + tokens[1], strings.Join(tokens[2:],":")

hmm, is that because the tag can themselves contain ":" in them?

Reply:
I am not sure if they are prevented

Reply:
That fails already:
```
$ ./bin/podman tag docker.io/library/ubuntu foo1:20000/foo2:foo3:foo4
Error: invalid reference format

lsm5 @ nagato : ~/go/src/github.com/containers/libpod(tag-correction) $ ./bin/podman tag docker.io/library/ubuntu foo1:20000/foo2:foo3     
lsm5 @ nagato : ~/go/src/github.com/containers/libpod(tag-correction) $ 
```

Reply:
Yes I checked the same thing, I guess we can let this go
lgtm

Reply:
err I made some more changes ... made len == 3 the default and <N/A> for len == 0. PTAL ...

Reply:
went back to the original change itself as lots of failures otherwise.